doc/source/contributing.md

```diff
@@ -0,0 +1,144 @@
+5. Someone from our team reviews the pull request for accuracy and if everything is correct, merges it into the documentation. The pull request may be redone due to internal requirements, such as the need to align the Percona branch to the MySQL branch. Then it gets published on the site at the next release.
+
+### Edit documentation locally
+
+This option is for users who prefer to work from their computer and/or have the full control over the documentation process.
+
+The steps are the following:
+
+1. Fork this repository
+2. Clone the repository on your machine:
+
+```sh
+git clone git@github.com:<your_name>/percona-server.git
+```
+
+3. Change the directory to ``percona-server/docs`` and add the remote origin repository:
+
+```sh
+git remote add origin git@github.com:percona/percona-server.git
+```
+
+4. Pull the latest changes from origin
+
+```sh
+git fetch origin
+git merge origin/<branch>
+```
+Make sure that your local branch and the branch you merge changes from are the same. So if you are on ``8.0`` branch, merge changes from ``origin/8.0``.
+
+5. Create a separate branch for your changes
+
+```sh
+git checkout -b <my_changes>
+```
+
+6. Make changes
+7. Commit your changes
+8. Open a pull request to Percona
+
+### Building the documentation
+
+To verify how your changes look, generate the static site with the documentation. This process is called *building*. You can do it in these ways:
+- [Use Docker](#use-docker)
+- [Install Sphinx and build locally](#install-sphinx-and-build-locally)
+
+#### Use Docker
+
+1. [Get Docker](https://docs.docker.com/get-docker/)
+2. We use [this Docker image](https://hub.docker.com/r/ddidier/sphinx-doc) to build documentation. Run the following command:
+
+```sh
+docker run --rm -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 make clean html
+```
+   If Docker can't find the image locally, it first downloads the image, and then runs it to build the documentation.
+
+3. Go to the ``doc/build/html`` directory and open the ``index.html`` file to see the documentation.
+4. Your static site will look different from the one on the web site. This difference is because we use a Percona theme that is rendered when the documentation is published on the web. To view the documentation with the Alabaster theme, run the following command:
+
+```sh
+docker run --rm -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 sphinx-build -b html -D 'html_theme=alabaster' source build/html
+```
+
+5. To create a PDF version of the documentation, run the following command:
+
+```sh
+docker run -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 make clean latex && docker run -i -v `pwd`:/doc -e USER_ID=$UID ddidier/sphinx-doc:0.9.0 make clean latexpdf
+```
+
+The PDF document is in the ``doc/build/latex`` folder.
```

Comment:
```suggestion
The resulting PDF document can be found in the ``doc/build/latex`` folder.
```

Reply:
What about troubleshooting? How can I check the documentation build for errors?
I think we should encourage contributors to verify their changes locally, to avoid they "break the build".

Reply:
Rewrote the section to include tips to build